gradle/dependencies.gradle

```
@@ -77,7 +77,7 @@ libraries.kryo =                [coordinates: 'com.esotericsoftware.kryo:kryo',
 libraries.maven3 =              [coordinates: 'org.apache.maven:maven-core', version: '3.0.5', license: "Apache 2.0"]
 libraries.maven3_wagon_file =   [coordinates: 'org.apache.maven.wagon:wagon-file', version: '3.0.0', because: '3.1.0 of wagon-http seems to break Digest authentication', license: "Apache 2.0"]
 libraries.maven3_wagon_http =   [coordinates: 'org.apache.maven.wagon:wagon-http', version: libraries.maven3_wagon_file.version, license: "Apache 2.0"]
-libraries.nativePlatform =      [coordinates: 'net.rubygrapefruit:native-platform', version: '0.22-milestone-3', license: "Apache 2.0"]
+libraries.nativePlatform =      [coordinates: 'net.rubygrapefruit:native-platform', version: '0.22-snapshot-20200615134452+0000', license: "Apache 2.0"]
```

```suggestion
libraries.nativePlatform =      [coordinates: 'net.rubygrapefruit:native-platform', version: '0.22-snapshot-20200615140910+0000', license: "Apache 2.0"]
```

Is this the version from your branch? Can you use `0.22-snapshot-20200615140910+0000` which is a snapshot from `master` where we archived the DLLs and PDB files?

---

Done.